src/cpp/CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ endif()
 
 if(OpenVINODeveloperPackage_FOUND)
     # must be called after all target_link_libraries
-    # ov_add_api_validator_post_build_step(TARGET ${TARGET_NAME})
+    ov_add_api_validator_post_build_step(TARGET ${TARGET_NAME})
```

Comment:
What is it for? I see that `ov_add_api_validator_post_build_step()` boils down to OneCore API validation, but I don't think genai uses OneCore

Reply:
This is to validate that GenAI is compatible with OneCore 

OpenVINO has such requirement from some users